From: Sophie Brun <sophie@freexian.com>
Date: Tue, 5 Nov 2019 14:13:31 +0100
Subject: Add Python 3.8 in Programming Language

Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=942753
Last-Update: 2019-11-05
---
 build_scripts/config.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build_scripts/config.py b/build_scripts/config.py
index 25036a6..f32dccb 100644
--- a/build_scripts/config.py
+++ b/build_scripts/config.py
@@ -93,6 +93,7 @@ class Config(object):
             'Programming Language :: Python :: 3.5',
             'Programming Language :: Python :: 3.6',
             'Programming Language :: Python :: 3.7',
+            'Programming Language :: Python :: 3.8',
         ]
 
         self.setup_script_dir = None
