From: Sophie Brun <sophie@freexian.com>
Date: Tue, 5 Nov 2019 10:39:14 +0100
Subject: Fix spelling errors

Last-Update: 2018-07-12

Last-Update: 2018-07-12
---
 sources/pyside2/libpyside/pysideproperty.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sources/pyside2/libpyside/pysideproperty.cpp b/sources/pyside2/libpyside/pysideproperty.cpp
index d9d15eb..e1e2c29 100644
--- a/sources/pyside2/libpyside/pysideproperty.cpp
+++ b/sources/pyside2/libpyside/pysideproperty.cpp
@@ -242,7 +242,7 @@ PyObject *qPropertySetter(PyObject *self, PyObject *callback)
         Py_INCREF(callback);
         return callback;
     }
-    PyErr_SetString(PyExc_TypeError, "Invalid property setter agument.");
+    PyErr_SetString(PyExc_TypeError, "Invalid property setter argument.");
     return nullptr;
 }
 
@@ -258,7 +258,7 @@ PyObject *qPropertyGetter(PyObject *self, PyObject *callback)
         Py_INCREF(callback);
         return callback;
     }
-    PyErr_SetString(PyExc_TypeError, "Invalid property getter agument.");
+    PyErr_SetString(PyExc_TypeError, "Invalid property getter argument.");
     return nullptr;
 }
 
@@ -365,7 +365,7 @@ int setValue(PySideProperty *self, PyObject *source, PyObject *value)
         Shiboken::AutoDecRef result(PyObject_CallObject(fset, args));
         return (result.isNull() ? -1 : 0);
     } else {
-        PyErr_SetString(PyExc_AttributeError, "Attibute read only");
+        PyErr_SetString(PyExc_AttributeError, "Attribute read only");
     }
     return -1;
 }
